src/rippled/lib/streams.js

```diff
@@ -44,6 +44,16 @@ const fetchLedger = (ledger, attempts = 0) => {
     });
 };
 
+const fetchServerInfo = () => {
+  return getServerInfo()
+    .then(result => result.info)
+    .then(info => {
```

Comment:
Can these two lines be combined into one? 

ie `.then(result => { info = result.info ... })`